dot/state/grandpa.go

```diff
@@ -194,7 +199,10 @@ func (s *GrandpaState) ApplyScheduledChanges(finalizedHeader *types.Header) erro
 	logger.Debugf("Applying authority set change scheduled at block #%d",
 		changeToApply.change.announcingHeader.Number)
 
-	// TODO(#3218): add afg.applying_scheduled_authority_set_change telemetry info here
+	canonHeightString := strconv.FormatUint(uint64(changeToApply.change.bestFinalizedNumber), 10)
```

Comment:
Hey, I take a look in the substrate code and I notice that the `change.canon_height` is the `announcingHeader.Number` which means the block where the scheduled/forced change takes place
 
ref: https://github.com/paritytech/substrate/blob/50de15d8740a129db9c18a6698fbd183b00326a2/client/consensus/grandpa/src/authorities.rs#L582

Reply:
Thanks, I changed it for both methods